outputs.tf

```diff
@@ -1,24 +1,24 @@
 
 output "ssh_private_key" {
-  description = "The private key for the SSH keypair"
+  description = "SSH Private key to use to connect to bastion and management hosts over SSH."
   value       = module.ssh.ssh_private_key
   sensitive   = false
 }
 output "bastion_public_ip" {
-  description = "The public IP address of the bastion host"
+  description = "The public IP address of the bastion host."
   value       = equinix_metal_device.bastion.access_public_ipv4
 }
-output "ssh_forward_command" {
-  description = "SSH port forward command to use to connect to the VCF GUIs"
-  value       = "ssh -L 8000:172.16.11.101:443 -L 8100:172.16.10.3:443 -i ${module.ssh.ssh_private_key} root@${equinix_metal_device.bastion.access_public_ipv4}"
-}
 output "management_public_ip" {
-  description = "The public IP address of the windows management host"
+  description = "The public IP address of the windows management host."
   value       = equinix_metal_device.management.access_public_ipv4
 }
-
 output "management_password" {
   description = "Randomly generated password used for the Admin accounts on the management host."
   sensitive   = true
-  value       = random_password.management.result
+  value       = "Management Host Admin password: ${random_password.management.result}\n"
```

Comment:
The prefix and newline will make it harder for folks to plumb this value into other steps.

Reply:
Would something like this work so that others could plumb {{value}} by itself?

```
value = random_password.management.result
valueMessage = format("Management Host Admin password: %s\n", value)
```

Reply:
Sounds good. I'll make it just the value and take Walt's advice to have a separate output later if we want to have explicit instructions.